asf_heat_pump_suitability/getters/base_getters.py

```diff
@@ -156,7 +156,8 @@ def list_obj_s3_location(location: str) -> list:
 
     return o
 
-def get_df_from_parquet_s3_path(path: str, **kwargs) -> pl.DataFrame:
+
+def get_pl_df_from_parquet_s3_path(path: str, **kwargs) -> pl.DataFrame:
```

Comment:
```suggestion
def get_df_from_parquet_s3_path(path: str, **kwargs) -> pl.DataFrame:
```
think this can stay as it was for readability/succinctness as it's covered in the return type